poonam/bootstrap/Bootstrap_assignment5/Assign_5bootstrap.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Assignment 5 Bootstrap</title>
+        <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.0/dist/css/bootstrap.min.css" integrity="sha384-B0vP5xmATw1+K9KRQjQERJvTumQW0nPEzvF6L/Z6nronJ3oUOFUFpCjEUQouq2+l" crossorigin="anonymous">
+    </head>
+    <body>
+        <div class="container mt-5">
+            <div class="row">
+                <div class="col">
+                    <ul class="nav">
+                        <li class="nav-item">
+                          <a class="nav-link" href="#">Home</a>
+                        </li>
+                        <li class="nav-item">
+                          <a class="nav-link" href="#">Employee</a>
+                        </li>
+                        <li class="nav-item">
+                          <a class="nav-link disabled" href="#">Employee Details</a>
+                        </li>
+                      </ul>
+                </div>
+            </div>
+            <div class="row mt-5">
+                <div class="col">
+                    <h3>List of Employees <span class="badge badge-info">New</span></h3>
+                    <table class="table">
+                        <thead class="bg-light">
+                          <tr>
+                            <th>Firstname</th>
+                            <th>Lastname</th>
+                            <th>Email</th>
```

Comment:
```suggestion
                            <th>Email</th>
```
Not Email
It's Mobile No
```suggestion
+<th>Mobile No</th>
```